### PR TITLE
Added support for json sclar value based on ValueNode<Dictionary<stri…

### PR DESCRIPTION
**Pull request by HakuTheWolfSpirit**
…ng, object>>

### CODE REVIEW

File: src/GraphQL.Conventions/Attributes/Execution/Wrappers/ObjectWrapper.cs [hunk 1]
```diff
@@ -16,7 +17,43 @@ public ObjectWrapper(IWrapper parent)
...
+            
+            Type valueType = value.GetType();
+            
+            bool isPrimitive = typeInfo.IsPrimitive;
+            isPrimitive |= valueType.GetTypeInfo().IsPrimitive;
+            isPrimitive |= value is string;
+            
+            shouldReturnValue |= isPrimitive;
+            
+            if(shouldReturnValue)
```
Comment (sungam3r) [inline src/GraphQL.Conventions/Attributes/Execution/Wrappers/ObjectWrapper.cs]:
```suggestion
            if (shouldReturnValue)
```

**Comment by HakuTheWolfSpirit** [comment]
@tlil 
image

**Comment by tlil** [comment]
LGTM @Patrick-Spiegel, but could you add some test coverage please?

**Comment by HakuTheWolfSpirit** [comment]
> LGTM @Patrick-Spiegel, but could you add some test coverage please?

Done. Sorry for the delay.

**Review by tlil — APPROVED** [review]

**Pull request merged**